packages/mutation-testing-elements/src/components/mutation-test-report-file/index.scss

```diff
@@ -17,12 +17,6 @@
     background-color: adjust-color($color: theme-color('caution'), $lightness: 40%);
 }
 
-code,
-code.hljs {
-    overflow: visible;
-    height: 100%;
-}
-
 pre {
-    overflow: visible;
-}
\ No newline at end of file
+    overflow-y: hidden !important;
```

Comment:
Do we realy need an `!important`? Also: why is overflow `hidden` (maybe add a comment to explain)?

Reply:
Changed it to `visible` now for the popup. 
![image](https://user-images.githubusercontent.com/10114577/73635472-ee2f3800-4663-11ea-8ac8-65c79ea2302d.png)

> Do we realy need an !important?

Unfortunately, prismjs has a very specific selector of `pre[class*="language-"]` which takes precedence over just `pre`. So it's either a more specific selector like `pre.line-numbers` (which I don't like because it is so specific) or an important. What do you think? Maybe an id?